### ISSUE TITLE
Confirmation Email not sent on plan change

### ISSUE THREAD

**Issue by pamelatech** [wid-tech/website#67]
**Describe the bug**
When a user is approved as a Community Member in a given chapter, there is a checkbox as the plan is created called "notify user".   When checked, one of two email templates should fire -- either the Membership Purchase template (if the user has never had this plan before) or the Membership Renewal template (if the user has been part of the plan before).   Currently no email is received by the user when this box is ticked.  

**What kind of user are you?**
User whose plan is changed by an administrator

**To Reproduce**
1. Use the Join Now link to register a new user
2. Open the user's home chapter, change their plan to "Community Member" and tick the notify user box. Click "Change" at the bottom of the page
3. Wait for an email
4. (nothing happens)

**Expected behavior**
User should get an email with the Membership Purchase template for that chapter

**Screenshots**
![image](https://user-images.githubusercontent.com/2591320/134780750-1abf5231-861c-4cb0-b500-34600c247116.png)

**Comment by mcresilkstart** [comment]
@pamelatech I think this is because the email send settings at the chapter level are restricted (I opened them up at the APEX but not the chapters).  I have just changed that to send all emails in the Global chapter.  Can you try your test again with someone joining GLobal as their home chapter and check if it now works.  If it still fails I will have to investigate further

**Comment by pamelatech** [comment]
I just tested again -  I think that instead of unrestricting the settings for Global, you have restricted them.  Now, I see the expected emails for the womeninidentity.org domain users but no messages for outside email addresses.  Previously, the outside email address could receive the welcome email from Global but not the membership plan.

So I believe we still have a problem, we need to unrestrict Global again to be sure.

**Comment by mcresilkstart** [comment]
@pamelatech I haven't changed any settings as they look to be set at both the APEX and in Global to all emails will be sent.  I created a test user, assigned to global, went to global and assigned that test user to the community member plan, ticked the 'notfiy user' box and saved.  I then checked I got the email and it arrived as expected as in below image.  So to me this looks to be working.

![image](https://user-images.githubusercontent.com/89020483/134807596-b141870d-1e48-4eb3-aa8b-0b10695c509d.png)
